README.md

```diff
@@ -105,8 +105,14 @@ Get an RDF dump from wikibase:
 
 ```docker-compose exec wikibase php ./extensions/Wikibase/repo/maintenance/dumpRdf.php```
 
+**Bash access**
+
+```docker exec -i -t wikibasedocker_wikibase_1 /bin/bash```
```

Comment:
You actually want to do:

docker-compose exec "wikibase" bash

as is used @ https://github.com/addshore/mediawiki-docker-dev/blob/master/bash#L6

The wikibasedocker_ prefix for the container depends on the directory of the docker compose file and shouldn't be depended on.

Reply:
okay